### PR TITLE
Medication use by place of death and month

### PR DESCRIPTION
**Pull request by stutibagri**

### CODE REVIEW

File: project.yaml [hunk 1]
```diff
@@ -18,6 +18,8 @@ actions:
       moderately_sensitive:
         monthly_deaths_place: output/os_reports/eol_service/deaths_month.csv
         monthly_deaths_place_plot: output/os_reports/eol_service/deaths_month_plot.png
+        eol_med_place: output/os_reports/eol_service/eol_med_pod.csv
```
Comment (stutibagri) [inline project.yaml:21]: The name can be changed to 'monthly_meds_place' and 'monthly_meds_place_plot' in the yaml file

Comment (NTMDavies) [reply project.yaml:21]: Agreed. I think consistency is good and will help us.

**Review by sophie-julian — APPROVED** [review]

File: project.yaml [hunk 2]
```diff
@@ -18,6 +18,8 @@ actions:
       moderately_sensitive:
         monthly_deaths_place: output/os_reports/eol_service/deaths_month.csv
         monthly_deaths_place_plot: output/os_reports/eol_service/deaths_month_plot.png
+        monthly_med_place: output/os_reports/eol_service/eol_med_pod.csv
```
Comment (eiliskeeble) [inline project.yaml:21]: These csv and png file names should ideally be consistent with the others

File: analysis/os_reports/01_eol_service_report_charts.R [hunk 3]
```diff
@@ -299,6 +299,45 @@ ggsave(deaths_month_plot, dpi = 600, width = 20, height = 10, unit = "cm"
        , filename = "deaths_month_plot.png"
        , path = here::here("output", "os_reports", "eol_service"))
 
+# Use of medications for symptom management  -------------------------------------------
+
+# Medication use by place of death - including all deaths
+eol_med_pod <- df %>%
```
Comment (eiliskeeble) [inline analysis/os_reports/01_eol_service_report_charts.R:305]: Dataframe name to be consistent with others eol_med_month

File: analysis/os_reports/01_eol_service_report_charts.R [hunk 4]
```diff
@@ -299,6 +299,45 @@ ggsave(deaths_month_plot, dpi = 600, width = 20, height = 10, unit = "cm"
...
+  group_by(study_month, pod_ons_new) %>%
+  summarise(mean = mean(eol_med_1m, na.rm = TRUE)) %>%
+  bind_rows(df %>%
+              group_by(study_month) %>%
+              summarise(mean = mean(eol_med_1m, na.rm = TRUE)) %>%
+              mutate(pod_ons_new = "All"))
+
+write_csv(eol_med_pod, here::here("output", "os_reports", "eol_service", "eol_med_pod.csv"))
```
Comment (eiliskeeble) [inline analysis/os_reports/01_eol_service_report_charts.R:313]: csv filename to be consistent with others

**Review by sophie-julian — APPROVED** [review]

File: analysis/os_reports/01_eol_service_report_charts.R [hunk 5]
```diff
@@ -299,6 +299,45 @@ ggsave(deaths_month_plot, dpi = 600, width = 20, height = 10, unit = "cm"
...
+                     , breaks = seq(0
+                                    , plyr::round_any(max(eol_med_pod$mean)
+                                                      , 1, f = ceiling)
+                                    , 1)
+                     , labels = scales::comma) +
+  NT_style() +
+  theme(axis.text.x = element_text(angle = 45, vjust = 1, hjust = 1))
+
+ggsave(eol_med_plot, dpi = 600, width = 20, height = 10, unit = "cm"
+       , filename = "eol_med_plot.png"
```
Comment (eiliskeeble) [inline analysis/os_reports/01_eol_service_report_charts.R:339]: png filename to be consistent with others

File: analysis/os_reports/02_eol_service_report.Rmd [hunk 6]
```diff
@@ -118,42 +118,45 @@ knitr::include_graphics(here::here("output", "os_reports", "eol_service", "gp_mo
...
 ## 4. Medications for symptom management
 
-People who died at home during the pandemic were more likely to be prescribed at least on medication for symptom management than people who died at home before the pandemic. 
+People who died at home during the pandemic were more likely to be prescribed at least one medication for symptom management than people who died at home before the pandemic. 
 
 ### Place of death
 
+_Figure 4: Average number of medications prescribed per person by month and place of death_
+```{r fig4, fig.height=1, fig.width=2, fig.cap=""}
+knitr::include_graphics(here::here("output", "os_reports", "eol_service", "eol_med_plot.png"))
```
Comment (eiliskeeble) [inline analysis/os_reports/02_eol_service_report.Rmd:127]: Filename will need updating when changed in other files

File: analysis/os_reports/01_eol_service_report_charts.R [hunk 7]
```diff
@@ -299,6 +299,45 @@ ggsave(deaths_month_plot, dpi = 600, width = 20, height = 10, unit = "cm"
...
+  group_by(study_month, pod_ons_new) %>%
+  summarise(mean = mean(eol_med_1m, na.rm = TRUE)) %>%
+  bind_rows(df %>%
+              group_by(study_month) %>%
+              summarise(mean = mean(eol_med_1m, na.rm = TRUE)) %>%
+              mutate(pod_ons_new = "All"))
+
+write_csv(eol_med_pod, here::here("output", "os_reports", "eol_service", "eol_med_pod.csv"))
+
+eol_med_plot <- ggplot(eol_med_pod, aes(x = study_month, y = mean
```
Comment (eiliskeeble) [inline analysis/os_reports/01_eol_service_report_charts.R:315]: this should be consistent with others eol_med_month_plot

File: analysis/os_reports/01_eol_service_report_charts.R [hunk 8]
```diff
@@ -299,6 +299,45 @@ ggsave(deaths_month_plot, dpi = 600, width = 20, height = 10, unit = "cm"
...
+
+# Medication use by place of death - including all deaths
+eol_med_pod <- df %>%
+  group_by(study_month, pod_ons_new) %>%
+  summarise(mean = mean(eol_med_1m, na.rm = TRUE)) %>%
+  bind_rows(df %>%
+              group_by(study_month) %>%
+              summarise(mean = mean(eol_med_1m, na.rm = TRUE)) %>%
+              mutate(pod_ons_new = "All"))
+
```
Comment (eiliskeeble) [inline analysis/os_reports/01_eol_service_report_charts.R:312]: Could you add a rounding step like for the other monthly dataframes created. It was added into the main branch after your branch was taken off.

**Pull request merged**